docs/metrics-3.6.md

```diff
@@ -0,0 +1,81 @@
+# Metrics upgrade notes
+
+Metrics have changed between 3.5 and 3.6.
+
+You can now retrieve metrics using the OpenTelemetry Protocol using the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/), and this is the recommended mechanism.
+
+These notes explain the differences in using the prometheus `/metrics` endpoint to scrape metrics for a minimal effort upgrade. It is not recommended you follow this guide blindly, the new metrics have been introduced because they add value, and so they should be worth collecting and using.
+
+## TLS
+
+The prometheus `/metrics` endpoint defaults to TLS on.
+
+To disable this set `.metricsConfig.secure` to `false`.
+
+## New metrics
+
+The following are new metrics:
+
+* `build_info`
+* `total_count`
+* `pods_total_count`
+* `controller_build_info`
+* `cronworkflows_triggered_total`
+* `workflowtemplate_triggered_total`
+* `workflowtemplate_runtime`
+* `k8s_request_duration`
+* `queue_duration`
+* `queue_longest_running`
+* `queue_retries`
+* `queue_unfinished_work`
+* `pod_pending`
+
+and can be disabled with
+
+```yaml
+metricsConfig:
```

Comment:
```suggestion
metricsConfig: |
```

must be a string since this is in a ConfigMap

Reply:
Yep, sorted.